src/server/debug.rs

```diff
@@ -386,6 +388,101 @@ impl Debugger {
         Ok(())
     }
 
+    pub fn remove_region(&self, region_id: u64) -> Result<()> {
+        let kv = self.engines.kv_engine.as_ref();
+        let raft = self.engines.raft_engine.as_ref();
+        let kv_wb = WriteBatch::new();
+        let raft_wb = WriteBatch::new();
+
+        let key = keys::raft_state_key(region_id);
+        let raft_state = match box_try!(raft.get_msg::<RaftLocalState>(&key)) {
+            Some(s) => s,
+            None => return Err(Error::Other("No RaftLocalState in RAFT rocksdb".into())),
+        };
+        box_try!(clear_meta(
```

Comment:
Why not just write a tombstone state? They will be cleared when tikv is started.

Reply:
In the original design `remove-region` is used for testing `init-empty-region`. If we just set it tombstone, we need to restart tikv twice for it.